pkg/arvo/sys/vane/ames.hoon

```diff
@@ -5074,6 +5110,8 @@
     |=  old=ames-state-15
     ^-  ^ames-state
     %=    old
+      cong  [cong.old `[~[/ames] /dead-flow `@da`(add now ~m2)]]
```

Comment:
How is this picked up and turned into a move during migration? Seems like it should be initialized to `~` here and then toggled in `+on-born`.

Reply:
It happens here: 

https://github.com/urbit/urbit/blob/82d4e2a265adb3aa3e1ab05a9c4365ae885eb026/pkg/arvo/sys/vane/ames.hoon#L1761-L1765

Reply:
I see, thanks.